src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/services/PDFGenerator.java

```diff
@@ -494,5 +501,11 @@ private String getLanguageString(String key) {
   private String getUserLanguage() {
     return (this.userProfile != null) ? this.userProfile.getProfileSettingPreference().getLanguage() : "nb";
   }
+
+  private boolean includePageInPdf(String layoutKey) {
```

Comment:
Could potentially consider moving this method out as a util function.

Reply:
Moved to LayoutUtils